styles/colors-titanium.less

```diff
@@ -101,4 +106,19 @@
 @agate-switch-bg-color: @agate-foreground;
 @agate-switch-selected-bg-color: @agate-accent;
 @agate-switch-color: @agate-title-text-color;
-@agate-switch-selected-color: @agate-title-text-color;
\ No newline at end of file
+@agate-switch-selected-color: @agate-title-text-color;
+
+
+// Toggle Colors
+// ---------------------------------------
+@agate-toggle-border-color: @agate-gray;
+@agate-toggle-indicator-border-color: transparent;
+@agate-toggle-focus-text-color: @agate-foreground-focus;
+@agate-toggle-button-switch-on-bg-color: @agate-highlight;
+@agate-toggle-button-switch-on-border-color: @agate-highlight;
+@agate-toggle-button-switch-on-focus-bg-color: @agate-white;
+@agate-toggle-button-switch-on-focus-border-color: @agate-white;
+@agate-toggle-button-switch-off-bg-color: @agate-gray;
+@agate-toggle-button-switch-off-border-color: @agate-white;
```

Comment:
I don't see this used anywhere. Can we remove it?

Reply:
done!